src/main/scala/week5/5-1-more-functions-on-lists.sc

```diff
@@ -1,30 +1,28 @@
-package week5
```

Comment:
@lbragaglia @liufengyun I have to remove packages from scala session files because otherwise interactive view does not work in inteliij idea